drodrpg/DRODLib/SettingsKeys.cpp

```diff
@@ -53,6 +53,7 @@ namespace Settings
 	DEF(DamagePreview);
 	DEF(DisableMouse);
 	DEF(DisplayCombos);
+	DEF(EnableAutosave);
```

Comment:
Is this setting hooked in to game logic anywhere?

Reply:
It looks like I didn't commit the changes for that. Oops!